TicTacToe/GameStatusChecker.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace TicTacToe
+{
+    public enum GameStatus
+    {
+        GameInProgress,
+        XWins,
+        OWins,
+        Draw
+    }
+
+    public static class GameStatusChecker
+    {
+        public static GameStatus CheckIfGameHasEnded(Field field)
+        {
+            return GameStatus.GameInProgress;
```

Comment:
Not sure I understand how two top tests pass with this line

Reply:
They do not pass. Functionality is not implemented. Trying TDD))